File: Unidad 4/p5_u4.cs
﻿using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int f;

            Console.WriteLine("Ingrese las frutas a procesar: ");
            f = Convert.ToInt32(Console.ReadLine());

            int[] frutas = new int[f];

            for (int i = 0; i < frutas.Length; i++)
            {
                Console.WriteLine("Escribe una fruta en indice " + i + ": ");
                frutas[i] = Convert.ToInt32(Console.ReadLine());
            }


        }
    }
}

File: Unidad 4/P5_u4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p5_u4;
import java.util.Scanner;
/**1
 * 
 *
 * @author richy
 */
public class P5_u4 {

    public static void main(String[] args) {
         Scanner sc=new Scanner (System.in); 
        int f;

            System.out.println("Ingrese las frutas a procesar: ");
            f = sc.nextInt();

            int[] frutas = new int[f];

            for (int i = 0; i < frutas.length; i++)
            {
                System.out.println("Escribe una fruta en indice " + i + ": ");
                frutas[i] = sc.nextInt();
            }


        }
    }
